### PR TITLE
chore: update release configuration

### PR DESCRIPTION
**Pull request by jonaslagoni**
**Description**
This PR updates the release configuration to use the newest CI system.

**Related issue(s)**
Fixes https://github.com/asyncapi/html-template/issues/429

### CODE REVIEW

**Comment by jonaslagoni** [comment]
@fmvilas @magicmatatjahu @derberg one of you have to add the correct tags as well :)

**Comment by derberg** [comment]
@jonaslagoni tags added

but this Pr do not fix the related issue, there is still one custom workflow that needs a fix - https://github.com/asyncapi/html-template/blob/master/.github/workflows/pr-testing-with-generator.yml (do you want to fix it in this PR or leave it for others?)

**Comment by jonaslagoni** [comment]
Ill fix it here

**Comment by derberg** [comment]
thanks, can you make sure we no longer use generator directly `npm i @asyncapi/generator` but AsyncAPI CLI through npx?

**Review by fmvilas — APPROVED** [review]

**Comment by jonaslagoni** [comment]
> thanks, can you make sure we no longer use generator directly `npm i @asyncapi/generator` but AsyncAPI CLI through npx?

Done 👍

File: .github/workflows/pr-testing-with-generator.yml [hunk 1]
```diff
@@ -12,14 +12,19 @@ jobs:
...
-          node-version: 14
+          node-version: "${{ steps.lockversion.outputs.version }}"
+          cache: 'npm'
+          cache-dependency-path: '**/package-lock.json'
       - name: Install dependencies
         run: NODE_ENV=production npm install
-      - name: Install generator
-        run: npm i @asyncapi/generator
+      - name: Install CLI
+        run: npm i -g @asyncapi/cli
```
Comment (derberg) [inline .github/workflows/pr-testing-with-generator.yml:28]:
```suggestion
```

File: .github/workflows/pr-testing-with-generator.yml [hunk 2]
```diff
@@ -12,14 +12,19 @@ jobs:
...
       - name: Run generation
-        run: node_modules/@asyncapi/generator/cli.js https://raw.githubusercontent.com/asyncapi/generator/v1.1.5/test/docs/dummy.yml ./ -o test/output --force-write
+        run: asyncapi generate fromTemplate https://raw.githubusercontent.com/asyncapi/generator/v1.1.5/test/docs/dummy.yml ./ -o test/output --force-write
```
Comment (derberg) [inline .github/workflows/pr-testing-with-generator.yml:30]:
```suggestion
        run: npx -p @asyncapi/cli asyncapi generate fromTemplate https://raw.githubusercontent.com/asyncapi/generator/v1.1.5/test/docs/dummy.yml ./ -o test/output --force-write
```

**Review by derberg — CHANGES_REQUESTED** [review]

**Review by derberg — APPROVED** [review]
Thanks!!!

**Pull request merged**